<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>ClientClass</class>
    <widget class="QMainWindow" name="ClientClass">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>798</width>
                <height>853</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Photo to 3D</string>
        </property>
        <widget class="QWidget" name="centralWidget"/>
        <widget class="QMenuBar" name="menuBar">
            <property name="geometry">
                <rect>
                    <x>0</x>
                    <y>0</y>
                    <width>798</width>
                    <height>21</height>
                </rect>
            </property>
            <widget class="QMenu" name="menuFile">
                <property name="title">
                    <string>File</string>
                </property>
                <addaction name="actionClearBackground"/>
                <addaction name="actionOpenImage"/>
                <addaction name="separator"/>
                <addaction name="actionClearSilhouette"/>
                <addaction name="actionLoadSilhouette"/>
                <addaction name="actionSaveSilhouette"/>
                <addaction name="separator"/>
                <addaction name="actionOpenCGA"/>
                <addaction name="actionSaveCGA"/>
                <addaction name="actionSaveOBJ"/>
                <addaction name="separator"/>
                <addaction name="actionExit"/>
            </widget>
<!--            <widget class="QMenu" name="menuOption">-->
<!--                <property name="title">-->
<!--                    <string>Tool</string>-->
<!--                </property>-->
<!--                <widget class="QMenu" name="menuRendering">-->
<!--                    <property name="title">-->
<!--                        <string>Rendering</string>-->
<!--                    </property>-->
<!--                    <addaction name="actionLineRendering"/>-->
<!--                    <addaction name="actionHatchingRendering"/>-->
<!--                    <addaction name="actionSSAORendering"/>-->
<!--                </widget>-->
<!--                <addaction name="actionBuildingReconstruction"/>-->
<!--                <addaction name="separator"/>-->
<!--                <addaction name="actionMassReconstruction"/>-->
<!--                <addaction name="actionAutoTest"/>-->
<!--                <addaction name="separator"/>-->
<!--                <addaction name="actionFacadeReconstruction"/>-->
<!--                <addaction name="separator"/>-->
<!--                <addaction name="actionOption"/>-->
<!--                <addaction name="menuRendering"/>-->
<!--            </widget>-->
            <widget class="QMenu" name="menuEdit">
                <property name="title">
                    <string>Edit</string>
                </property>
                <addaction name="actionUndo"/>
            </widget>
            <addaction name="menuFile"/>
            <addaction name="menuEdit"/>
<!--            <addaction name="menuOption"/>-->
        </widget>
        <widget class="QToolBar" name="mainToolBar">
            <attribute name="toolBarArea">
                <enum>TopToolBarArea</enum>
            </attribute>
            <attribute name="toolBarBreak">
                <bool>false</bool>
            </attribute>
        </widget>
        <widget class="QStatusBar" name="statusBar"/>
        <action name="actionOpenImage">
            <property name="text">
                <string>Open Image</string>
            </property>
        </action>
        <action name="actionExit">
            <property name="text">
                <string>Exit</string>
            </property>
        </action>
        <action name="actionMassReconstruction">
            <property name="text">
                <string>Mass Reconstruction</string>
            </property>
            <property name="shortcut">
                <string>Ctrl+P</string>
            </property>
        </action>
        <action name="actionNew">
            <property name="text">
                <string>New</string>
            </property>
            <property name="shortcut">
                <string>Ctrl+N</string>
            </property>
        </action>
        <action name="actionOpenCGA">
            <property name="text">
                <string>Open CGA</string>
            </property>
        </action>
        <action name="actionLoadSilhouette">
            <property name="text">
                <string>Load Silhouette</string>
            </property>
            <property name="shortcut">
                <string>Ctrl+O</string>
            </property>
        </action>
        <action name="actionSaveSilhouette">
            <property name="text">
                <string>Save Silhouette</string>
            </property>
            <property name="shortcut">
                <string>Ctrl+S</string>
            </property>
        </action>
        <action name="actionUndo">
            <property name="text">
                <string>Undo</string>
            </property>
            <property name="shortcut">
                <string>Ctrl+Z</string>
            </property>
        </action>
        <action name="actionExtractCameraParameter">
            <property name="text">
                <string>Extract Camera Parameter</string>
            </property>
        </action>
        <action name="actionParameterEstimationWithCameraCalibration">
            <property name="text">
                <string>Parameter Estimation with Camera Calibration</string>
            </property>
        </action>
        <action name="actionParameterDialog">
            <property name="text">
                <string>Parameter Dialog</string>
            </property>
        </action>
        <action name="actionSaveScreen">
            <property name="text">
                <string>Save Screen</string>
            </property>
        </action>
        <action name="actionOption">
            <property name="text">
                <string>Option</string>
            </property>
        </action>
        <action name="actionPenVanishingLine">
            <property name="checkable">
                <bool>true</bool>
            </property>
            <property name="text">
                <string>Vanishing Line</string>
            </property>
        </action>
        <action name="actionPenSilhouette">
            <property name="checkable">
                <bool>true</bool>
            </property>
            <property name="text">
                <string>Silhouette</string>
            </property>
        </action>
        <action name="actionClearLines">
            <property name="text">
                <string>Clear Lines</string>
            </property>
        </action>
        <action name="actionSaveLines">
            <property name="text">
                <string>Save Lines</string>
            </property>
        </action>
        <action name="actionLoadLines">
            <property name="text">
                <string>Load Lines</string>
            </property>
        </action>
        <action name="actionClearSilhouette">
            <property name="text">
                <string>Clear Silhouette</string>
            </property>
        </action>
        <action name="actionClearBackground">
            <property name="text">
                <string>Clear Background</string>
            </property>
        </action>
<!--        <action name="actionAutoTest">-->
<!--            <property name="text">-->
<!--                <string>Auto Test</string>-->
<!--            </property>-->
<!--        </action>-->
<!--        <action name="actionFacadeReconstruction">-->
<!--            <property name="text">-->
<!--                <string>Facade Reconstruction</string>-->
<!--            </property>-->
<!--        </action>-->
<!--        <action name="actionTextureMapping">-->
<!--            <property name="text">-->
<!--                <string>Texture Mapping</string>-->
<!--            </property>-->
<!--        </action>-->
<!--        <action name="actionLineRendering">-->
<!--            <property name="checkable">-->
<!--                <bool>true</bool>-->
<!--            </property>-->
<!--            <property name="text">-->
<!--                <string>Line Rendering</string>-->
<!--            </property>-->
<!--        </action>-->
<!--        <action name="actionSSAORendering">-->
<!--            <property name="checkable">-->
<!--                <bool>true</bool>-->
<!--            </property>-->
<!--            <property name="text">-->
<!--                <string>SSAO</string>-->
<!--            </property>-->
<!--        </action>-->
<!--        <action name="actionHatchingRendering">-->
<!--            <property name="checkable">-->
<!--                <bool>true</bool>-->
<!--            </property>-->
<!--            <property name="text">-->
<!--                <string>Hatching</string>-->
<!--            </property>-->
<!--        </action>-->
<!--        <action name="actionBuildingReconstruction">-->
<!--            <property name="text">-->
<!--                <string>Building Reconstruction</string>-->
<!--            </property>-->
<!--        </action>-->
        <action name="actionSaveOBJ">
            <property name="text">
                <string>Save OBJ</string>
            </property>
        </action>
        <action name="actionSaveCGA">
            <property name="text">
                <string>Save CGA</string>
            </property>
        </action>
        <action name="actionLoadSilhouetteOld">
            <property name="text">
                <string>Load Silhouette Old</string>
            </property>
        </action>
    </widget>
    <layoutdefault spacing="6" margin="11"/>
    <resources>
        <include location="Client.qrc"/>
    </resources>
    <connections/>
</ui>
